apitools/base/py/base_api.py

```diff
@@ -398,6 +399,14 @@ def max_retry_wait(self, value):
                 'max_retry_wait must be a postiive integer')
         self.__max_retry_wait = value
 
+    @property
```

Comment:
Sure. I was just trying to keep things uniform, as the check_response_func property also had a getter/setter.  I agree that they're unnecessary; making both of them public properties.